File: Harvard_CS50_Introduction_to_CS/CS50-Pset6-master/mario_more.py
# import get_int from cs50
from cs50 import get_int
# ask user to input the positive height of the pyramid
height = 0
while(height < 1 or height > 8):
    height = get_int("Height: ")
# print pyramid
for i in range(1, height + 1):
    # print space
    print(" " * (height - i), end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i)

File: Harvard_CS50_Introduction_to_CS/CS50-Pset1-master/mario_more.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = 0;
    int x = 0;
    while (!(n > 0 && n < 9))
    {
        n = get_int("Height: ");
    }
    for (int i = 1; i < n+1 ;i++)
    {
        while (x >= 0 && x < n-i)
        {
            printf(" ");
            x++;
        }
        while (x >= n-i && x < n)
        {
            printf("#");
            x++;
        }
        printf("  ");
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
        x = 0;
    }
}
